components/DesignSystemProvider.tsx

```diff
@@ -1,4 +1,6 @@
 import React from 'react';
-import { IdProvider } from '@radix-ui/react-id';
+import { TooltipProvider } from '@radix-ui/react-tooltip';
 
-export const DesignSystemProvider: React.FC = ({ children }) => <IdProvider>{children}</IdProvider>;
+export const DesignSystemProvider: React.FC = ({ children }) => (
+  <TooltipProvider>{children}</TooltipProvider>
+);
```

Comment:
If we do this, everyone who uses the DS would have the `Tooltip` in their bundle even if they're not using it. Is that something we're concerned about?

Reply:
It shouldn't be an issue for us at Modulz, all our apps use almost all of the components.

Reply:
Yeah, this is mainly for us